doc/release_notes.rst

```diff
@@ -3,6 +3,32 @@ Release Notes
 =============
 
 
+Release 1.1.0
+=============
+New functionality and features
+------------------------------
+* Merged TSPE "Total spiking probability edges" into elephant, a tool for classifying excitatory and inhibitory synaptic effects using connectivity estimation (https://github.com/NeuralEnsemble/elephant/pull/560).
```

Comment:
I think this should be written more from a user's perspective and less technical
```suggestion
* New method "Total spiking probability edges" (TPSE) for inferring functional connectivity (https://github.com/NeuralEnsemble/elephant/pull/560).
```

Reply:
I agree.